Themis/Views/Exercises/ExerciseSections.swift

```diff
@@ -12,46 +12,44 @@ struct ExerciseDateProperty: Hashable {
     let dateKeyPath: KeyPath<Exercise, String?>
 }
 
-struct ExercisesListView: View {
+struct ExerciseSections: View {
```

Comment:
I don't see a reason , why to change the name. If we change it, I would at least prefer that it is called ExerciseSectionsView.